web/src/components/FeedbackButtons.tsx

```diff
@@ -29,8 +29,18 @@ const FeedbackButtons = ({ rating, setRating }: FeedbackButtonsProps): ReactElem
     <>
       <div>{t('description')}</div>
       <StyledToggleButtonGroup exclusive value={rating} onChange={handleChange}>
-        <ToggleButton value={RATING_POSITIVE} icon={SentimentSatisfiedOutlinedIcon} text={t('useful')} />
-        <ToggleButton value={RATING_NEGATIVE} icon={SentimentDissatisfiedOutlinedIcon} text={t('notUseful')} />
+        <ToggleButton
+          iconSize='medium'
```

Comment:
🔧 Since we unified the icon size, there is no more `small` and therefore no more need to have this option/prop. Please remove it.